<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1586</width>
    <height>898</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>2</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-image:url(&quot;D:\我的桌面\b站自学\pythonProject2\Resource\Pic\v2-12c09440b7049d539b5e9cd86665d48a_1440w.jpg&quot;)
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="Pic_frame">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>150</y>
      <width>501</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { border: 2px solid black; }</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>-1</number>
    </property>
    <widget class="QLabel" name="PIC_Show">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>480</width>
       <height>360</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    border: 2px solid gray;
    border-radius: 5px;
    padding: 5px;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Resource/Myqrc.qrc">:/path/Pic/伞-小车.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="margin">
      <number>1</number>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>461</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="save_pic">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #5DA5DA;
    border-radius: 5px;
    border: 2px solid #1B4F72;
    color: white;
    font-weight: bold;
    font-size: 16px;
    padding: 5px 10px;
}

QPushButton:hover {
    background-color: #1B4F72;
    border-color: #5DA5DA;
    cursor: pointer;
}
</string>
        </property>
        <property name="text">
         <string>保存图片</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_suspend">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #5DA5DA;
    border-radius: 5px;
    border: 2px solid #1B4F72;
    color: white;
    font-weight: bold;
    font-size: 16px;
    padding: 5px 10px;
}

QPushButton:hover {
    background-color: #1B4F72;
    border-color: #5DA5DA;
    cursor: pointer;
}
</string>
        </property>
        <property name="text">
         <string>停止播放</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="Low_crl_Frame">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>80</y>
      <width>311</width>
      <height>411</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QScrollBar" name="verticalScrollBar_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>20</width>
       <height>361</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>258</width>
       <height>381</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="order_low">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;下位机指令&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignJustify|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QTextBrowser {
    border: 2px solid gray;
    border-radius: 5px;
    padding: 5px;
}
</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="UDP_frame">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>371</width>
      <height>601</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <widget class="QPushButton" name="Run_model">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>510</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>开始运行</string>
     </property>
    </widget>
    <widget class="QLabel" name="UDP_send">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>440</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>UDP发送</string>
     </property>
    </widget>
    <widget class="QLabel" name="UDP_recieve">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>170</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>UDP接收</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="mssg_rec">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>104</y>
       <width>141</width>
       <height>210</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>109</width>
       <height>210</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QTextBrowser {
    border: 2px solid gray;
    border-radius: 5px;
    padding: 5px;
}
</string>
     </property>
     <property name="markdown">
      <string/>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="placeholderText">
      <string>这里将会显示接受的信息</string>
     </property>
    </widget>
    <widget class="QLabel" name="Upper_ipLB">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>70</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>上位机IP</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mssg_sed">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>440</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
 {
    border: 2px solid gray;
    border-radius: 5px;
    padding: 5px;
}
</string>
     </property>
     <property name="placeholderText">
      <string>请输入信息</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Upper_ipEd">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
 {
    border: 2px solid gray;
    border-radius: 5px;
    padding: 5px;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>上位机IP</string>
     </property>
    </widget>
    <widget class="QLabel" name="Lower_ipED">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>360</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    border: 2px solid gray;
    border-radius: 5px;
    padding: 5px;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="Lower_ipLB">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>360</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>1</width>
       <height>1</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>3</width>
       <height>3</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>3</width>
       <height>3</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>下位机IP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Sendmssg">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>440</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>570</y>
      <width>311</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    background-color: white;
    border: 2px solid gray;
    border-radius: 5px;
    margin-top: 1ex;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    color: rgb(16, 72, 255);  /* 修改标题的颜色为青绿色 */
    padding: 0 3px;
}
</string>
    </property>
    <property name="title">
     <string>下位机控制</string>
    </property>
    <widget class="QDial" name="dial">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QDial {
    background-color: #f5f5f5;
    border: 1px solid #ddd;
    border-radius: 50%;
    color: #333;
    font-size: 14px;
    font-family: &quot;Microsoft Yahei&quot;;
    padding: 10px;
    margin: 20px;
}

</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Right">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resource/Myqrc.qrc">
       <normaloff>:/path/Icon/caret-right.png</normaloff>:/path/Icon/caret-right.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="Left">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resource/Myqrc.qrc">
       <normaloff>:/path/Icon/caret-left.png</normaloff>:/path/Icon/caret-left.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="Up">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushBotton{
	background-image:url(:/path/Icon/caret-up.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resource/Myqrc.qrc">
       <normaloff>:/path/Icon/caret-up.png</normaloff>:/path/Icon/caret-up.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="Down">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resource/Myqrc.qrc">
       <normaloff>:/path/Icon/caret-down.png</normaloff>:/path/Icon/caret-down.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1700</width>
      <height>1000</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>2000</width>
      <height>1000</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resource/Myqrc.qrc">:/path/Pic/img1.wallspic.com-chou_xiang_yi_shu-jian_yue-qi_fen-sheng_tai_qu-azure-3840x2160.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>Pic_frame</zorder>
   <zorder>Low_crl_Frame</zorder>
   <zorder>UDP_frame</zorder>
   <zorder>groupBox</zorder>
  </widget>
 </widget>
 <tabstops>
  <tabstop>save_pic</tabstop>
  <tabstop>Upper_ipEd</tabstop>
  <tabstop>mssg_sed</tabstop>
  <tabstop>Run_model</tabstop>
  <tabstop>btn_suspend</tabstop>
  <tabstop>mssg_rec</tabstop>
  <tabstop>Up</tabstop>
  <tabstop>Left</tabstop>
  <tabstop>Down</tabstop>
  <tabstop>Right</tabstop>
 </tabstops>
 <resources>
  <include location="Resource/Myqrc.qrc"/>
  <include location="C:/Users/13959/.designer/backup/Resource/Myqrc.qrc"/>
 </resources>
 <connections/>
</ui>
